### ISSUE TITLE
Accessibility audit - TagGroup

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#820]
accessibility audit PDF:
https://inovaciebratislava.slack.com/archives/C06FW05TXHC/p1730379593029059

![Image](https://github.com/user-attachments/assets/fe87a4f6-490a-4dd9-a41d-f921aac38546)

![Image](https://github.com/user-attachments/assets/07d26b77-e1b3-46ad-8096-ee213dc7aded)

**Comment by katarinazaprazna** [comment]
Re-check with documentation and Teplický

**Comment by katarinazaprazna** [comment]
- Response from Peter Teplický - the component has a11y issues
- This component should be revisited for accessibility
